settings_test.py

```diff
@@ -1,4 +1,4 @@
-from settings import *
+from settings import *  # noqa
```

Comment:
Between this and @robhudson, I declare the Age of Peppering the Codebase With "noqa" has begun. Long live NOQA!

(I don't have any better ideas. I'm just kvetching.)